deepchecks/utils/plot.py

```diff
@@ -158,3 +169,51 @@ def shifted_color_map(cmap, start=0, midpoint=0.5, stop=1.0, name: str = 'shifte
 def hex_to_rgba(h, alpha):
     """Convert color value in hex format to rgba format with alpha transparency."""
     return 'rgba' + str(tuple([int(h.lstrip('#')[i:i+2], 16) for i in (0, 2, 4)] + [alpha]))
+
+
+def create_confusion_matrix_figure(confusion_matrix, x, y, is_normalize):
+    """Create a confusion matrix figure.
+
+    Parameters
+    ----------
+    confusion_matrix: np.ndarray
+        array containing the confusion matrix.
+    x: np.ndarray
+        array containing x axis data.
+    y: np.ndarray
+        array containing y axis data.
+    is_normalize: bool
+        if True will also show normalized values by the true values.
+
+    Returns
+    -------
+    go.Figure
+        confusion matrix figure
+
+    """
+    if is_normalize:
```

Comment:
If the values are between 0 and 100 it's not commonly referred to as "normalization". Though I guess Bressler asked for it to be in percentages and not in [0,1] range?
I don't have any other suggestion though.